app/controllers/api/functions.php

```diff
@@ -447,13 +451,14 @@
     ->desc('Create Deployment')
     ->label('scope', 'functions.write')
     ->label('event', 'functions.[functionId].deployments.[deploymentId].create')
+    ->label('audits.resource', 'function/{request.functionId}/deployment/{response.$id}')
```

Comment:
I think we can save all deployments events on the main function resource no need to nest down.